Makefile

```diff
@@ -127,15 +123,11 @@ dev-push:
 # Create YAML file for deployment
 prod-yaml: | $(KUSTOMIZE)
 	@$(KUSTOMIZE) version 2>&1 | grep -q 'KustomizeVersion:v\{0,1\}[2-9][[:digit:]]\{0,\}' || { echo "kustomize v2+ required" 1>&2; exit 1; }
-	@echo "updating kustomize image patch file for manager resource"
-	sed -i'' -e 's@image: .*@image: '"${PRODUCTION_IMG}"'@' ./config/default/vsphere_manager_image_patch.yaml
-	cmd/clusterctl/examples/vsphere/generate-yaml.sh
+	VSPHERE_MANAGER_IMG=${PRODUCTION_IMG} cmd/clusterctl/examples/vsphere/generate-yaml.sh
```

Comment:
Hi @andrewsykim,

If there's nothing left in the target but this line, then why don't you move the kustomize version check to `generate-yams.sh`?